docs/section10/10-10.md

```diff
@@ -0,0 +1,26 @@
+# Airside System - Rule 10-10  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False  
+**Rule ID:** 10-10  
+**Rule Description:** Where no heating system exists or no heating system has been submitted with design documents, the proposed building system type shall be the same system as modeled in the baseline building design and shall comply with but not exceed the requirements of Section 6.  
+**Rule Assertion:** P-RMR = B-RMR   
+**Appendix G Section:** Table G3.1 Section 10 proposed column c.   
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Evaluate each HeatingVentilationAirConditioningSystem Data Group
+
+**Applicability Checks:** None
+
+**Function Calls:**  
+1. get_proposed_hvac_modeled_with_virtual_heating()
+
+## Rule Logic:  
+**Applicability Check 1:**  
+- Call function to get list of applicable HVAC systems due to virtual heating being modeled in the proposed per G3.1-10 c:`applicable_HVAC_systems_heating_list_p = get_proposed_hvac_modeled_with_virtual_heating(U_RMR,P_RMR)`
+- For each hvac system in applicable_HVAC_systems_heating_list_p: `for hvac_p in applicable_HVAC_systems_heating_list_p:`    
+    **Rule Assertion:**
+    - Case 1: For applicable HVAC system: `outcome == "UNDETERMINED" and raise_message "Check that the proposed building system type is modeled the same system as is modeled in the baseline building design and that it complies with but does not exceed the requirements of Section 6."`"`  
```

Comment:
Please updated message to indicate that the heating system should be checked, not just the "proposed building system".

Reply:
Done